### ISSUE TITLE
modify_target Error: HOSTS requires an EXCLUDE_HOSTS

### ISSUE THREAD

**Issue by sarritzu** [greenbone/python-gvm#187]
Hello,

just to let you know that the function "modify_target" requires the "exclude_hosts" parameter if you set the "hosts" parameter.  Even if it's supposed to be optional...

e.g. you will get that error if you try to call

`gmp.modify_target(target_id, hosts=['192.168.1.2'])`

you will get the error:

`gvm.errors.GvmError: ('Error in response. HOSTS requires an EXCLUDE_HOSTS', <Element modify_target_response at 0x7fe668f3f588>)`

In order to avoid the error you have to use instead:

`gmp.modify_target(target_id, hosts=['192.168.1.2'], exclude_hosts=[''])`

**Comment by bjoernricks** [comment]
@y0urself do you have some time to validate this issue and implement a fix? Something like

```python3
if hosts and exclude_hosts is None:
  exclude_hosts = ['']
```

**Comment by y0urself** [comment]
Sure

**Comment by bjoernricks** [comment]
Thanks!

**Comment by y0urself** [comment]
If I tested this bug correctly, it appears when I use the `gmp.modify_target()` method and also if I directly send a `<modify_target>` request over ssh.
I have done a workaround for the `gmp` class, but it will **not** help, when you use raw xml requests - these need to look like this:

```xml
<modify_target target_id='fancy_id'>
  <name>really good name</name>
  <hosts>some,hosts,here</hosts>
  <exclude_hosts/>
</modify_target>
```

or

```xml
<modify_target target_id='fancy_id'>
  <name>really good name</name>
  <hosts>some,hosts,here</hosts>
  <exclude_hosts></exclude_hosts>
</modify_target>
```

**Comment by bjoernricks** [comment]
Should be fixed with  #188
